tests-functional/config.json

```diff
@@ -7,12 +7,15 @@
     "HTTPEnabled": true,
     "HTTPHost": "0.0.0.0",
     "HTTPPort": 3333,
-    "HTTPVirtualHosts": ["*", "status-go"],
+    "HTTPVirtualHosts": [
+        "*",
+        "status-go"
+    ],
     "APIModules": "eth,admin,wallet,accounts,waku,wakuext,ethclient",
     "WalletConfig": {
-      "Enabled": true
+        "Enabled": true
     },
-    "WakuConfig": {
+    "WakuV2Config": {
         "Enabled": false
```

Comment:
should be `true` now? 🤔 
though I think this file is simple not used at all. cc @antdanchenko

Reply:
Good point but functional-tests are green. Seems unused indeed.